src/components/DropMenu/DropMenu.less

```diff
@@ -0,0 +1,75 @@
+.lucid-DropMenu {
+
+	&.lucid-DropMenu-base {
+		display: inline-block;
+	}
+
+	.lucid-DropMenu-Control {
+		position:relative;
+		outline: 0px;
+	}
+
+	.lucid-DropMenu-Option {
+		cursor: pointer;
+		padding: 0 10px;
+		line-height: @Button-size-lineHeight;
+		font: @font;
+		height: 25px;
+		display: flex;
+		align-items: center;
+
+		&.lucid-DropMenu-Option-is-disabled {
+			color: @color-disabledText;
+			font-style: italic;
+			cursor: not-allowed;
+		}
+
+		&.lucid-DropMenu-Option-is-focused {
+			background: @color-lightGray
+		}
+
+		&.lucid-DropMenu-Option-is-selected {
+			background: @color-primaryMedium
+		}
+
+		&.lucid-DropMenu-Option-is-null {
+			color: @color-disabledText;
+		}
+	}
+
+	&.lucid-ContextMenu-FlyOut {
+		#normalize > .fix-chrome-scrollbar-zindex();
+		.box-sizing();
+		background-color: @color-white;
+		border: solid 1px @featured-color-default-borderColor;
+		overflow-y: auto;
+		max-height: 420px; // arbitrary
```

Comment:
Move this into a variable at the top of the file called `@DropMenu-size-maxHeight`